packages/property-grid/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bentley/property-grid-react",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
did you manually change this? if so revert this, and run `rush change`

Reply:
oh right, thanks!